health/health.d/ping.conf

```diff
@@ -0,0 +1,49 @@
+
+ template: ping_host_reachable
+ families: *
+       on: ping.host_packet_loss
+    class: Errors
+     type: Other
```

Comment:
Should type be `System` ? (as in `ping_packet_loss`) ?

Reply:
It is "type: Other" because I copied fping alarms and adjusted them. If you ask me - none of them should be System if System refers to the host system (which I am not sure about).

Reply:
Changed ping_packet_loss to Other.